Add new BBEs (#3092)
ballerina-platform/ballerina-distribution: examples/spread-in-binding-patterns/spread_in_binding_patterns.md

> # Spread in binding patterns

Reviewer: Instead of spread, shall we say

```suggestion
# Rest binding pattern in binding patterns
```

Reviewer: Or just 
```suggestion
# Rest binding pattern
```

> You can use the spread operator in mapping, list, and error binding patterns.

Reviewer: ```suggestion
You can use the rest binding pattern in mapping, list, and error binding patterns.
```

IMO, we need to improve the description to say what the rest binding pattern actually does. Similar to what we've specified for rest in match.

> You can use the spread operator in binding patterns for mappings, lists and errors.

Reviewer: ```suggestion
You can use the spread operator in binding patterns for mappings, lists, and errors.
```

> You can use the spread operator in binding patterns for mappings, lists, and errors.

Reviewer 2: ```suggestion
You can use the `spread` operator in binding patterns for mappings, lists, and errors.
```

> You can use the `spread` operator in binding patterns for mappings, lists, and errors.

Reviewer: We emphasize only keywords, identifiers, etc.

```suggestion
You can use the spread operator in mapping, list, and error binding patterns.
```

> You can use the rest binding pattern (...r) to bind the fields that are not explicitly bound in the binding pattern. This can be used in mapping, list, and error binding patterns.

Reviewer: ```suggestion
You can use the rest binding pattern (`...r`) to bind the fields that are not explicitly bound in the binding pattern. This can be used in mapping, list, and error binding patterns.
```
